eng/versioning/external_dependencies.txt

```diff
@@ -116,7 +116,7 @@ org.postgresql:postgresql;42.3.8
 org.slf4j:slf4j-api;1.7.36
 org.slf4j:slf4j-nop;1.7.36
 org.slf4j:slf4j-simple;1.7.36
-redis.clients:jedis;4.2.3
+redis.clients:jedis;4.3.1
```

Comment:
Oh no. :( I'll revert it then.

Reply:
It's not because of your change. The original version is not compatible. But I think if we ship a library with Jedis dependency, we'd better pin the version to one compatible with Spring Boot 2.

The Jedis version compatible with Spring Boot 2 is 
```xml
<jedis.version>3.8.0</jedis.version>
```

Reply:
Thank you for clarifying